Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -812,6 +895,17 @@ define([
                     if (that.frustums) {
                         that.frustumStatisticText = frustumStatsToString(that._scene.debugFrustumStatistics);
                     }
+                    if (that.globeDepth) {
+                        // Determine the number of frustums being used.
+                        var numberOfFrustums = that._scene._frustumCommandsList.length;
```

Comment:
Don't reach into private members of `Scene` here.  Instead add a (perhaps `@private`) getter to `Scene`.